.changeset/config.json

```diff
@@ -14,6 +14,7 @@
   ],
   "snapshot": {
     "useCalculatedVersion": true,
-    "prereleaseTemplate": "{tag}-{datetime}-{commit}"
+    "prereleaseTemplate": "alpha-{commit}",
```

Comment:
This does ths whole commit -- I saw suggestions for `shortCommit` or `{commit:6}` but neither of these worked. I think this should be fine, just not elegant ig